libminifi/include/utils/file/FileUtils.h

```diff
@@ -80,6 +80,16 @@ class FileUtils {
    * @param path current path to delete
    * @param delete_files_recursively deletes recursively
    */
+
+  static char get_separator()
+  {
+#ifdef _WIN32
```

Comment:
Why the difference in guards ?

Reply:
Just copied from a code of mine, aligned to the file, thanks!